Dockerfile-frankyapp

```diff
@@ -21,6 +21,10 @@ ENV DOCKER_NETWORK_HOST=0.0.0.0
 
 COPY --from=builder /install /usr/local
 COPY ./backend /home/runner/franky/backend
+
+
+# COPY ./backend/private-key.pem /home/runner/franky/backend/private-key.pem
```

Comment:
Probably, this is some debug staff which can be excluded from the final docker file.

Reply:
+++